client/my-sites/checkout/composite-checkout/components/wp-order-review-line-items.js

```diff
@@ -487,10 +489,17 @@ function returnModalCopy(
 				  );
 			break;
 		case 'domain':
-			modalCopy.title = translate( 'You are about to remove your domain from the cart' );
-			modalCopy.description = translate(
-				'When you press Continue, we will remove your domain from the cart and you will have no claim for the domain name you picked.'
-			);
+			if ( isRenewal ) {
+				modalCopy.title = translate( 'You are about to remove your domain renewal from the cart' );
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.